tests/unit/test_tempest_extremes.py

```diff
@@ -577,6 +577,33 @@ def test_stitch_nodes_threshold_filters(
         assert result["stderr"] == "Mocked stderr output"
         assert result["returncode"] == 0
 
+    def test_stitch_nodes_values_from_dn(self) -> None:
```

Comment:
Consistent naming in function name, and explicit.
```suggestion
    def test_stitch_nodes_values_from_detect_nodes(self) -> None:
```